src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -242,21 +241,22 @@ public boolean isOperational()
     @Override
     public int compareTo(Bridge o)
```

Comment:
Do we have any tests for this compareTo?  If not that would be a good thing to add.

Reply:
I've added some.